web/src/components/UI/button.tsx

```diff
@@ -0,0 +1,29 @@
+import * as React from 'react';
+
+export interface IButtonProps {
+    children?: React.ReactNode,
+    onClick?: (e: any) => void
+};
+
+const styles = {
```

Comment:
my suggestion here is to leverage rebass w/ emotion over a custom button component. Rebass has a button component that you can customize. 

```js
import { Button as RebassBtn} from 'rebass';
import styled from '@emotion/styled';

export const Button = styled(RebassBtn)`
...styles here (keep in mind these are true css properties)
`;


That or you can leverage `emotion-themeing` and create Button variants which will more closely accomplish what you have here.